nemo/collections/asr/models/label_models.py

```diff
@@ -170,10 +169,10 @@ def __setup_dataloader_from_config(self, config: Optional[Dict]):
                 normalize_audio=config.get('normalize_audio', False),
             )
 
-        if type(dataset) is ChainDataset:
-            collate_ds = dataset.datasets[0]
-        else:
+        if hasattr(dataset, 'fixed_seq_collate_fn'):
```

Comment:
The others above were fine, but this is very brittle. Do a better check.

Reply:
Made it consistent with others.